src/onboarding/constants.js

```diff
@@ -0,0 +1,8 @@
+var PROPERTIES_TYPE_SCRIPT = 1;
+var PROPERTIES_TYPE_USER = 2;
+var PROPERTIES_TYPE_DOCUMENT = 3;
+
+var PROFILE_IMAGE_FILE_ID = "1KvZudA9OtE-PRPyY7kP567kjhWGJqUop";
```

Comment:
these should not be here, should get retrieved from PropertiesService (script properties)

Reply:
They are all in the script properties. I run setProperties() function in onScriptCreated() function. This constants is just for developer's convience.

Reply:
They are all in the script properties. I run setProperties() function in onScriptCreated() function. This constants is just for developer's convience.